_pages/en/web-services-access.md

```diff
@@ -1,141 +1,185 @@
+Finally, it's worth noting that some Web tools can already be accessed on the ESDC network and are being used by employees (See [Appendix B](#appendix-b---web-tools-already-accessible)).
 
 ### Considerations
 
-More departments are using SaaS to collaborate and engage with citizens and other public administrations in Canada and the world.
-For example, the latest Open Government Plan and the Service and Digital Target Architecture from TBS were shared as drafts for consultation using Google Docs.
+Departments are leveraging Web tools to collaborate and engage with citizens and public administrations in Canada and the world.
+For example, the latest Open Government Plan and Service and Digital Target Architecture from TBS were shared as drafts for consultation and comments on Google Docs.
 The GC Digital Academy and Free Agents, as well as other groups, frequently use Google Forms for registration.
 
+In order to move forward, we propose a balanced approach in granting access to additional Web tools (See [Appendix A](#appendix-a---web-tools-to-grant-access)) only for IITB employees in the short term, as a step toward broader access in ESDC.
+As IITB employees generally don't deal with protected information and are well informed and trained on the management of information, we anticipate the risk to be minimal.
+On the other side, the benefits should bring great value such as enhancing collaboration and efficiency, increasing job satisfaction and helping attract and retain IT talent.
+
 ### Recommendations
 
-To comply to the new Policy on Service and Digital, ESDC needs to:
+To support the adoption of the new [Policy on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603), ESDC needs to:
 
-- Enable additional SaaS tools (See [Appendix A](#appendix-a---saas-to-unblock)) for IITB employees as a first step toward providing more open access to the Internet for more groups of ESDC employees.
-- Evaluate security monitoring tools to mitigate risks associated with more open access to the Internet SaaS tools.
-- Review its technology brick criteria by considering the shared responsibility model of SaaS
+- Grant access to additional Web tools (see [Appendix A](#appendix-a---web-tools-to-grant-access)) for IITB employees, as a step toward broader open access to the Internet for ESDC employees; and
+- Perform an evaluation of security monitoring tools, active at the source (application and database) and the perimeter (DLP), to mitigate future risks associated with broader open access to the Internet.
 
 ### Next Steps
 
-Should you approve, the IT Strategy team will work with teams in IITB to:
+The IT Strategy team will work with teams in IITB to:
+
+- Ensure the following Web tools (see [Appendix A](#appendix-a---web-tools-to-grant-access)) are functional on the network for IITB employees;
+- Ensure IITB employees' understanding of document classification;
```

Comment:
and that the tools and services are only to be used with unclassified information and data.

Reply:
would also add: work in conjunction with information management to ensure...